cmd/kubectl-kcpforedgesyncer/cliplugins/base/options.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
This file has the same contents as the corresponding one in kcp.

(1) Would it be better to import the existing package from kcp than to make a copy?

(2) If we do want to make a copy, would it be better to put it in `pkg/`, analogous to where the original sits in kcp and following the usual pattern?

Reply:
I didn't notice that the contents are same.
Fixed to import the package from kcp instead of copy&paste.